### PR TITLE
chore: Setup Testing Environment

### PR DESCRIPTION
**Pull request by maximeish**
### What this PR does
Add Jest and Sinon as testing frameworks

### Task to be completed
Modified package.json

### Manual testing
clone github repository then check package.json

### Background context
N/A

### Relevant pivotal tracker stories
https://trello.com/c/DjJg31l0/25-setup-testing-environment

### Screenshots
N/A

### CODE REVIEW

**Comment by julesntare** [comment]
[Setup Testing Environment](https://trello.com/c/DjJg31l0/25-setup-testing-environment)

**Review by IRANKUND — COMMENTED** [review]
well done Maxime

**Review by giranezafiacre — APPROVED** [review]
well done

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "orcas-phantom-frontend",
+  "version": "1.0.0",
+  "description": "Phantom is a web application that allows passengers to track bus location and movements around Kigali",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (malaba6) [inline package.json]: Plz, add the test scripts here

**Review by malaba6 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "orcas-phantom-frontend",
+  "version": "1.0.0",
+  "description": "Phantom is a web application that allows passengers to track bus location and movements around Kigali",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest ./src/test/"
```
Comment (maximeish) [inline package.json]: @Malaba6 
I added the test for Jest.
I thought we are only going to use Jest for testing, do we need Sinon too?

Comment (maximeish) [reply package.json:7]: @Malaba6 
I thought we are only going to use Jest for testing, do we need Sinon too?

**Review by julesntare — APPROVED** [review]

**Review by malaba6 — APPROVED** [review]
LGTM

**Comment by maximeish** [comment]
[Setup Testing Environment](https://trello.com/c/DjJg31l0/25-setup-testing-environment)

**Pull request merged**